contributors

```diff
@@ -18,6 +19,7 @@ The following contributors helped to bring check_pgactivity :
   * macarbiter
   * Frédéric Yhuel
   * Julian Vanden Broeck
+  * Benoit Lobréau
```

Comment:
I think that Shangzi Xie should also be added here.

Reply:
Well, he actually reported the issue on pgsql-bloat-estimation repository. I did the fix there, then in here. So, should it be considered as a contribution to check_pga?

Reply:
OK, indeed, I forgot Shangzi Xie actually studied the code to find this issue, then reviewed my brainless patch.